SMLHelper/Logger.cs

```diff
@@ -1,36 +1,80 @@
 ﻿namespace SMLHelper.V2
 {
     using System;
+    using System.IO;
+
+    internal enum LogLevel
+    {
+        Debug,
+        Warn,
+        Error,
+        Info
```

Comment:
The order should be Debug, Info, Warn, Error

Reply:
Info should always be displayed. In order to do that, it should be last.

Reply:
On that same note, Errors should always appear too. I am resolving this conversation for now,  I will add another review regarding this.

Reply:
Hmm, that is very true, perhaps if anything below Error is set, it can be set back to Error?